services/report/3rdparty/jasper-cs-report/src/main/resources/coreIntake.jrxml

```diff
@@ -21,19 +24,13 @@ inc.entryDate entryDate,
 inc.returnDate returnDate,
 h1.name AS "csid",
 co.objectnumber AS "objectnumber",
-case when (co.fieldcollectionplace  is not null and co.fieldcollectionplace  <> '') then
- regexp_replace(co.fieldcollectionplace , '^.*\)''(.*)''$', '\1')
- else ''
-end AS site,
+co.fieldcollectionplace AS site,
 ong.objectName AS "objectname",
 case when (bd.item is not null and bd.item <> '') then
  regexp_replace(bd.item, '^.*\)''(.*)''$', '\1')
  else ''
 end AS description,
```

Comment:
@mikejritter Do you know what's going on with this description field? It's still being de-urned in SQL. I suspect the reports that do this came from Berkeley, and they probably have some museum that uses this field to store a ref name.

Reply:
I looked through the Berkeley configs and I don't see anywhere where they changed briefDescription to hold a ref name. So I think we can get rid of this? It appears in a couple other reports too.

Reply:
I went ahead and removed the de-urning for the briefDescription fields. 

I had also tried on some of the remaining fields which use the regexp_replace logic (e.g. donor in coreAcquisition) but they weren't being de-urned when running the report. They all exist in the `<title>` section, and I took a brief look at the jasper scriplet API but didn't see any methods for that part of the lifecycle. Doesn't seem like a big deal atm since it's still being handled in the query.

Reply:
That sounds fine, thanks!